clusterloader2/pkg/measurement/common/slos/windows_node_resource_usage.go

```diff
@@ -99,6 +119,90 @@ func convertToMemoryPerfData(samples []*model.Sample) *measurementutil.PerfData
 	return perfData
 }
 
+func convertToStoragePerfData(samples []*model.Sample) *measurementutil.PerfData {
+	perfData := &measurementutil.PerfData{Version: currentWindowsResourceUsageMetricsVersion}
+	for _, sample := range samples {
+		item := measurementutil.DataItem{
+			Data: map[string]float64{
+				"Storage_Used": math.Round(float64(sample.Value)*100/(1024*1024*1024)) / 100,
+			},
+			Unit: "GB",
+			Labels: map[string]string{
+				"Volume": string(sample.Metric["volume"]),
+			},
+		}
+		perfData.DataItems = append(perfData.DataItems, item)
+	}
+	return perfData
+}
+
+func convertToOpenFilesPerfData(samples []*model.Sample) *measurementutil.PerfData {
+	perfData := &measurementutil.PerfData{Version: currentWindowsResourceUsageMetricsVersion}
+	for _, sample := range samples {
+		item := measurementutil.DataItem{
+			Data: map[string]float64{
+				"Open File Handles": math.Round(float64(sample.Value)),
+			},
+			Unit: "Files",
```

Comment:
This handle count is Unit of `Files` and below the hands are `Handles`.  Should this be consistent with Unit of `Handles`